app/components/nav-bar.js

```diff
@@ -8,5 +8,21 @@ export default Component.extend({
       this.get('authManager').logout();
       this.get('routing').transitionTo('index');
     }
+  },
+
+  didInsertElement() {
+    this._super.call(this);
+    this.$('.notification.item').popup({
+      popup : '.popup',
+      on    : 'click'
+    });
+  },
+
+  willDestroyElement() {
```

Comment:
@niranjan94 I am not sure, this is the correct way to go. Have tried to implement what I read at : 
https://medium.com/@chrisdmasters/cleaning-up-after-components-in-ember-js-73bbf0f16add